docs/teal-basics.md

```diff
@@ -0,0 +1,121 @@
+###### tags: `tealgarden` `ethturin`
+
+# Basic Explainers Teal Garden Track :books:
+
+**Hackers**, welcome to our quick explainers, to give you a conception of what you need to consider building with us at **ETH Turin**.
+
+Don't feel overwhelmed by all the materials, we simply want to give you a little primer to get started more easily :)
+
+## What is holacracy?
+
+**Holacracy** is a new way of structuring and running your organization that replaces the conventional management hierarchy. Instead of operating top-down, power is distributed throughout the organization – giving individuals and teams freedom while staying aligned to the organization’s purpose.
+
+**Holacracy-powered organizations** focus on purpose at every level of scale: organizational purpose, team purpose, and individual purpose are all explicit and aligned. Every team member directs their energy in alignment with the broader mission, unlocking your organization’s full potential.
```

Comment:
maybe add a link to further information. either the holacracy constitution, or a list of organizations that practice holacracy, so that the reader can give the concept some credability and aknowledge it as proven concept, rather than thinking that it is some pie in the sky idea.